spot-ml/install_ml.sh

```diff
@@ -1,12 +0,0 @@
- #!/bin/bash
```

Comment:
We have been talking about possibly creating packages, having for example parcels for cloudera would allow this code to be deployed as part of a node role